src/components/editor/MonacoEditor.tsx

```diff
@@ -5,24 +5,28 @@ import * as monacoEditor from 'monaco-editor/esm/vs/editor/editor.api';
 import { useRef } from 'react';
 import { useIntl } from 'react-intl';
 
-interface Props {
+export interface Props {
```

Comment:
I am an advocate for grouping optional properties and having all required properties listed first.